apps/airtable-cache/README.md

```diff
@@ -0,0 +1,35 @@
+# airtable-cache
+
+A caching proxy to Airtable. Consumers can generally ignore the details, and just pretend this is Airtable itself for standard [Airtable API](https://airtable.com/developers/web/api/) routes. For example, with [`airtable-ts`](https://github.com/domdomegg/airtable-ts):
```

Comment:
Need to explain why this exists, i.e.:

Airtable has low rate limits
Airtable is slow (sometimes)